trl/trainer/grpo_config.py

```diff
@@ -69,14 +69,14 @@ class GRPOConfig(TrainingArguments):
             `per_device_train_batch_size * num_processes * gradient_accumulation_steps`.
         steps_per_generations: (`int` or `None`, *optional*, defaults to `None`):
             Number of optimization steps per generation. If `None`, it defaults to gradient_accumulation_steps.
-        temperature (`float`, defaults to `0.9`):
+        temperature (`float`, defaults to `1.0`):
             Temperature for sampling. The higher the temperature, the more random the completions.
         top_p (`float`, *optional*, defaults to `1.0`):
             Float that controls the cumulative probability of the top tokens to consider. Must be in (0, 1]. Set to
             `1.0` to consider all tokens.
-        top_k (`int` or `None`, *optional*, defaults to `50`):
+        top_k (`int` or `None`, *optional*, defaults to `None`):
```

Comment:
Do we need to map `None` to `-1` in vLLM's `SamplingParams` of `grpo_trainer.py`? It would be good to check whether `None` is an allowed parameter or not in that case and whether it gives the same behaviour as `top_k=-1`

Reply:
Ah nevermind, saw we do this automatically here: https://github.com/huggingface/trl/blob/3ef9faf2574e9ead0edad668de8d41d228a088c9/trl/trainer/grpo_trainer.py#L1021